homeassistant/components/input_text/__init__.py

```diff
@@ -294,5 +294,12 @@ def state_attributes(self):
 
     async def async_update_config(self, config: typing.Dict) -> None:
         """Handle when the config is updated."""
+        # set defaults for empty config
+        config = {
```

Comment:
I wonder if the better fix here is to just run it through the voluptuous schema again? 


Should this be done for all the other input_* too?